benchmarks/config/matmul/1024x1024x512.json

```diff
@@ -0,0 +1,87 @@
+[
+  {
+  "matmul_1024x1024x512_fp32_dnn": {
+    "matmul_fp32_single_dnn": {
+      "type": "XSMM-DNN",
+      "benchmark": "xsmm_dnn_mlp",
+      "environment": { "OMP_NUM_THREADS": "1" },
+      "flags": [ "100", "1024", "0", "F", "64", "64", "64", "0", "512", "1024" ],
```

Comment:
aren't those all flipped?

Reply:
And measured numbers seems to align for DNN and MLIR in cases such as 128x768x3072 and 128x3072x768.